components/Novel.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+
+export default ({ title, author }) => (
+  <
+    div className="novel">{'${title} by ${author}'}</div>
```

Comment:
Remember that you need to use backticks instead of quotes to do string interpolation like this

```JSX
<div className="novel">{`${title} by ${author}`}</div>
```